.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "shared-modules"]
```

Comment:
Seems this isn't proper git submodule. Mind re-adding it with `git submodule add`?

Reply:
Thanks, I'll fix this.